src/ResourceManager/Compute/Commands.Compute/Generated/Disk/Config/NewAzureRmDiskUpdateConfigCommand.cs

```diff
@@ -59,26 +59,31 @@ public class NewAzureRmDiskUpdateConfigCommand : Microsoft.Azure.Commands.Resour
         [Parameter(
             Mandatory = false,
             ValueFromPipelineByPropertyName = true)]
+        [Obsolete("This parameter is obsolete.  It cannot be used during update.", false)]
```

Comment:
What is the alternative?  Please clarify what the user should do instead of using this paramater, to make this message truly useful.  This applies to all the Obsolete messages